### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by mohamed20163858**
update website style to work as well on desktop devices by transferring Figma desktop design rules inside my project, I used media queries to transform this knowledge and I changed all the images inside my website with ones that have a suitable big size for laptop and desktop dimensions beside I updated the rules of the grid, flexbox, and font-size to make my website has a nice visual view in big size devices like  laptop or desktop devices

### CODE REVIEW

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @mohamed20163858  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

- [ ] Please add a description to your pull request. You can list what changes you made in this pull request. 
- [x] For a viewport width between 768px and 1000px your website scrolls horizontally. That's because you have got a content wider than the size of the viewport. Please try to avoid a horizontal scroll by setting the width of your elements using `vw` and `%`. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -124,55 +129,61 @@ <h3>Multi-Post Stories<span>Gain+Glory</span></h3>
...
         </section>
-        <section class="aboutme">
+        <section class="aboutme" id="about">
             <div class="sticker">
                 <img src="images/crytalUp.png" alt="crystal-up-icon" class="crytal-1">
-                <h2>About me</h2>
-                <p> I can help you build a product , feature or website Look through some of my work and experience! If
-                    you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                <button class="button-style">Get My Resume</button>
+                <img src="images/crytalUPDesktop.png" alt="crystal-up-icon" class="crytal-1-desktop">
```
[the remark this answers is not in the document]
Comment (mohamed20163858) [reply index.html]: @wuletawwonte thanks a lot man i fixed it and the implementation was really awesome with it

File: css/main.css [hunk 2]
```diff
@@ -371,3 +407,238 @@ header a {
...
+  .section-1 {
+    background-image: url(../images/backgroundDesktop.png);
+    min-height: 1000px;
+    justify-content: flex-start;
+    align-items: center;
+  }
+
+  .section-1 .p1,
+  .section-1 .p2 {
+    width: 920px;
```
[the remark this answers is not in the document]
Comment (mohamed20163858) [reply css/main.css]: @wuletawwonte yes i do it and it helped me a lot to make my website more responsive thanks to you to mention that to me thanks a lot

**Review by omarsalem7 — APPROVED** [review]
Hi @mohamed20163858 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**